subcommands/waves/init.go

```diff
@@ -126,6 +148,29 @@ func doInitWave(cmd *cobra.Command, args []string) {
 	}
 }
 
+func pruneTargets(currentTargets *client.AtsTargetsMeta, versions []string) client.AtsTargetsMeta {
+	var missing []string
+	for _, version := range versions {
+		found := false
+		for name, file := range currentTargets.Targets {
+			custom, err := api.TargetCustom(file)
+			subcommands.DieNotNil(err)
+			if custom.Version == version {
```

Comment:
Using `slices.Contains(versions, custom.Version)` would allow to simplify it to just one loop here.